liquibase-core/src/main/java/liquibase/resource/ResourceAccessor.java

```diff
@@ -178,15 +178,15 @@ default Resource get(String path) throws IOException {
         } else if (resources.size() == 1) {
             return resources.iterator().next();
         } else {
-            String message = "Found " + resources.size() + " files with the path '" + path + "':" + System.lineSeparator();
+            final StringBuilder message = new StringBuilder("Found " + resources.size() + " files with the path '" + path + "':" + System.lineSeparator());
```

Comment:
Hi @arturobernalg,

Is there a specific reason for using concatenation as the value passed to the StringBuilder constructor, instead of creating a builder with a shorter string and then applying concatenations for the remaining parts? 

Thanks,
Daniel.

Reply:
I guess he did that as this is not part of the loop. But could have used the .append too